package.json

```diff
@@ -23,12 +23,24 @@
     "jquery": "^3.4.1",
     "react": "^16.4.2",
     "react-dom": "^16.4.2",
-    "react-scripts": "1.1.4",
+    "react-scripts": "^3.0.1",
     "styled-components": "^3.4.2",
     "uuid": "^3.1.0"
   },
   "devDependencies": {
     "backstopjs": "^3.8.5"
   },
-  "heroku-run-build-script": true
+  "heroku-run-build-script": true,
+  "browserslist": {
+    "production": [
+      ">0.2%",
+      "not dead",
+      "not op_mini all"
+    ],
+    "development": [
```

Comment:
And this.

Reply:
Resuming here:
> When compiling to development, add CSS vendor prefix to last versions of Chrome, Firefox and Safari.

You can read more about this in [browserlist README.md](https://github.com/browserslist/browserslist/blob/master/README.md) or in this nice article: [Outdated browser detection with Browserslist - dev.to](https://dev.to/amplifr/outdated-browser-detection-with-browserslist-10co)